nixos/configuration.nix

```diff
@@ -46,13 +46,7 @@
 
   systemd = {
     # https://github.com/NixOS/nixpkgs/blob/8e5e5a6add04c7f1e38e76f59ada6732947f1e55/nixos/doc/manual/release-notes/rl-2411.section.md?plain=1#L69-L76
-    # Disabling to avoid `SC2174` in for '/nix/store/h93h6srxzslr8kyv13klrq63zd6ymhxy-unit-script-cups-pre-start.drv'
-    enableStrictShellChecks = false;
-    services.systemd-suspend.environment = {
-      # Apply workaround as https://github.com/systemd/systemd/issues/33083#issuecomment-2142473011 to fix GH-959
-      # TODO: Check again after systemd 257 or later. At least updating to nixos-25.05
-      SYSTEMD_SLEEP_FREEZE_USER_SESSIONS = "false";
```

Comment:
https://github.com/kachick/dotfiles/issues/959#issuecomment-2533264029 seems to be fixed with this update, so dropped the workaround